src/scope.ts

```diff
@@ -1213,12 +1213,17 @@ export class ScopeScanner {
         } else {
             if (ts.isPropertyAssignment(node.parent)) {
                 functionName = node.parent.name.getText();
+                functionScope.envParamLen++;
```

Comment:
This function `_generateFuncScope` is designed to generate default functions, whose envParamLen is 1, for object literal's functions, we regard it as class methods now, whose envParamLen is 2.

Would it be better that delete this `if (ts.isPropertyAssignment(node.parent))` condition, but add this condition into here: https://github.com/intel/Wasmnizer-ts/blob/202df8b6e3d1ebdb66618211b70f25bfd5f80543/src/scope.ts#L967-L971
And if this condition is true, just invoke `_generateClassFuncScope`, and if false, invoke `_generateFuncScope`?